Forest.cs

```diff
@@ -24,27 +24,35 @@ public Forest(int ForestWidth, int ForestHeight)
                 // Row
                 for (int j = 0; j < forest.GetLength(0); j++)
                 {
-                    forest[j, i] = DecideTree(rng);
+                    forest[j, i] = DecideTree(rng, i + j + 1);
```

Comment:
(As we've talked) this works, but is a lil confusing. Might need a comment

Reply:
```suggestion
                    // Adds the indexes of the two dimensions of the array (+1)
                    // Gets how far into the 2D array an element is   
                    forest[j, i] = DecideTree(rng, i + j + 1);
```